runtime/doc/lsp.txt

```diff
@@ -2292,6 +2292,29 @@ is_enabled({bufnr})                      *vim.lsp.document_color.is_enabled()*
         (`boolean`)
 
 
+==============================================================================
+Lua module: vim.lsp.linked_editing_range            *lsp-linked_editing_range*
+
```

Comment:
should the module `@brief` mention what "linked editing session" is? Not obvious to me